lib/bolt/transport/docker.rb

```diff
@@ -8,7 +8,7 @@ module Bolt
   module Transport
     class Docker < Base
       def self.options
-        %w[host service-url service-options tmpdir interpreters]
+        %w[host service-url service-options tmpdir interpreters shell_command tty]
```

Comment:
Note that `service-url` has a hyphen, this probably should too for consistency in the config file.